templates/vud_widget.html.twig

```diff
@@ -0,0 +1 @@
+{%  include '{{ base_path }}/{{ widget_name }}/{{ widget_template }}.twig.html' %}
```

Comment:
In D7 this is implemented as:
```php
$template_file = $plugin['path'] . '/' . $plugin[$template_type . ' template'] . $plugin['extension'];
```
where:
* ctools defined by default `path` (vud did not), which was the path to the directory to where in our case files like `widgets/plain/plain.inc` lived, e.g. `sites/all/modules/vote_up_down/widgets/plain`. An extract from  D7 ctools documentation:
  > The path to the file containing the plugin. This is useful for using secondary files, such as templates, css files, images, etc, that may come with a plugin.
* `$plugin[$template_type . ' template']` ended up in either ` $plugin['widget template']` or ` $plugin['votes  template']`, e.g. `widget`.
* `$plugin['extension']`, which was defined on `vud_width_plugin_defaults()` to `.tpl.php`

With these the template file ended up, e.g. on plain widget as `sites/all/modules/vote_up_down/widgets/plain/widget.tpl.php`.

Can you modify this to actually get the right path? E.g. widget directories are not on the base_path (drupal core's `/`) AFAIK, but maybe I am wrong, can you provide an example of base_path, is this resolving correctly?